wear/src/main/java/com/example/k_dual/watchface/KDualCanvasRender.kt

```diff
@@ -44,6 +59,8 @@ class KDualCanvasRender (
 
     private val textBounds = Rect()
 
+    // TODO. paint 여기서 한번만 불러 오는 게 낫나?
```

Comment:
[ChatGPT] **Singleton for Paint Objects**: To reduce the creation of `Paint` objects during every render call, you could use a singleton pattern to create these objects once and reuse them. This is how you could implement a singleton for a `Paint` object.
```kotlin
object CustomPaints {
    val clockPaint by lazy {
        Paint().apply {
            color = Color.WHITE
            textSize = unitF * 15
            typeface = robotoMedium
            isAntiAlias = true
        }
    }
    // Add other paint objects here
}

// Usage in the render function:
val clockPaint = CustomPaints.clockPaint
```